website/src/content/blog/biome-v1-7.md

```diff
@@ -0,0 +1,306 @@
+This command will read your _ESLint_ configurations and attempt to port their settings to Biome.
+
+The command handles both the legacy and the flat _ESLint_ configurations.
+It supports the `extends` field of the legacy configuration and loads both shared and plugin configurations!
+The command also migrates `.eslintignore`.
+
+Given the following _ESLint_ configuration:
+
+```json title=".eslintrc.json"
+{
+  "extends": ["plugin:unicorn/recommended"],
+  "plugins": ["unicorn"],
+  "ignore_patterns": ["dist/**"],
+  "globals": {
+    "Global1": "readonly"
+  },
+  "rules": {
+    "eqeqeq": "error"
+  },
+  "overrides": [
+    {
+      "files": ["tests/**"],
+      "rules": {
+        "eqeqeq": "off"
+      }
+    }
+  ]
+}
+```
+
+And the following _Biome_ configuration (obtained by running `biome init`):
+
+```json title="biome.json"
+{
+	"organizeImports": {
+		"enabled": true
+	},
+	"linter": {
+		"enabled": true,
+		"rules": {
+			"recommended": true
+		}
+	}
+}
+```
+
+Run `biome migrate eslint --write` to migrate your _ESLint_ configuration to _Biome_.
+This results in the following _Biome_ configuration:
+
+```json title="biome.json"
+{
+	"organizeImports": { "enabled": true },
+	"linter": {
+		"enabled": true,
+		"rules": {
+			"recommended": false,
+			"complexity": {
+				"noForEach": "error",
+				"noStaticOnlyClass": "error",
+				"noUselessSwitchCase": "error",
+				"useFlatMap": "error"
+			},
+			"style": {
+				"noNegationElse": "off",
+				"useForOf": "error",
+				"useNodejsImportProtocol": "error",
+				"useNumberNamespace": "error"
+			},
+			"suspicious": {
+				"noDoubleEquals": "error",
+				"noThenProperty": "error",
+				"useIsArray": "error"
+			}
+		}
+	},
+	"javascript": { "globals": ["Global1"] },
+	"overrides": [
+		{
+			"include": ["tests/**"],
+			"linter": { "rules": { "suspicious": { "noDoubleEquals": "off" } } }
+		}
+	]
+}
+```
+
+Note that this overrides your initial _Biome_ configuration.
+The command requires _Node.js_ to be installed to load _JavaScript_ configurations such as `eslint.config.js` and to resolve the `extends` field.
+For now, `biome migrate eslint` doesn't support configuration written in _YAML_.
+
+We have a [dedicated page](/linter/rules-sources/) that lists the equivalent _Biome_ rule of a given _ESLint_ rule.
+We handle some _ESLint_ plugins such as _TypeScript ESLint_, _ESLint JSX A11y_, _ESLint React_, and _ESLint Unicorn_.
+Some rules are equivalent to their _ESLint_ counterparts, while others are inspired.
+By default, _Biome_ doesn't migrate inspired rules.
+You can use the _CLI_ flag `--include-inspired` to migrate them.
+
+## Migrate from _Prettier_ with a single command
+
+[_Biome v1.6_ introduced the command `biome migrate prettier`](/blog/biome-v1-6/#easier-migration-from-prettier).
+
+This new version adds the support of the `overrides` field and attempts to convert `.prettierignore` glob patterns to globs supported by _Biome_.
```

Comment:
```suggestion
In Biome v1.7, we add support of `overrides` field and attempts to convert `.prettierignore` glob patterns to globs supported by _Biome_.
```

Can you add a link to this overrides field? It isn't clear if we are talking about the prettier one or Biome one.

Plus, what does "support" mean? We should explain what the command does now with overrides, that's the place where we should talk about it.